test/mapmatch.cc

```diff
@@ -44,10 +44,11 @@ float round_up(float val, int multiple) {
   return int((val + multiple - 1) / multiple) * multiple;
 }
 
-std::string
-to_locations(const std::vector<PointLL>& shape, const std::vector<float>& accuracies, int frequency) {
+std::string to_locations(const std::vector<PointLL>& shape,
+                         const std::vector<float>& accuracies,
+                         int frequency = 0,
+                         int start_time = 8 * 60 * 60) {
```

Comment:
when generating locations for these tests its nice to be able to set a start time that isnt 0